src/haconiwa/task/submit.py

```
@@ -0,0 +1,628 @@
+    """Task submission error"""
+    pass
+
+
+class TaskSubmitter:
+    """Submit tasks to existing companies with automatic worktree creation"""
+    
+    def __init__(self, space_manager: SpaceManager = None, task_manager: TaskManager = None):
+        self.space_manager = space_manager or SpaceManager()
+        self.task_manager = task_manager or TaskManager()
+    
+    def submit_task(
```

Pylint (3.3.7)

[refactor] Too many arguments (12/5)

(R0913)